server/views/pages/start/legal-sender-start-page.test.ts

```diff
@@ -21,12 +20,12 @@ describe('Legal Sender Start Page View', () => {
     expect($('#legal-sender-start').length).toStrictEqual(1)
   })
 
-  it('should show supported prison names', () => {
-    viewContext = { prisonNames: ['some-prison', 'another-prison'] }
-
-    const $ = cheerio.load(compiledTemplate.render(viewContext))
-
-    expect($('[data-qa=prisonNames]').html()).toContain('some-prison')
-    expect($('[data-qa=prisonNames]').html()).toContain('another-prison')
+  it('should show updated any prison supported text instead of prison names', () => {
+    const $ = cheerio.load(compiledTemplate.render({}))
+    const startNowHrefSelector = 'a[href="/oneTimeCode/request-code"]'
```

Comment:
If you add an attribute to the relevant `h2` in the `.njk` file (e.g. `<h2 class="govuk-heading-m" data-qa="prisons">Prisons you can send mail to</h2>`) then you could just target it directly like this:

```javascript
expect($('[data-qa="prisons"]').text()).toEqual('Prisons you can send mail to')
expect($('[data-qa="prisons"] + p').text()).toContain(
  'You can use this service to send legal or confidential mail to any prison'
)
```

([cheerio](https://cheerio.js.org) uses jQuery style selectors so you don't necessarily need to add custom attributes if there's already a class or id you can target.)

Reply:
Updated both page and test based on your recommendations. Thanks.